spaceship/lib/spaceship/spaceauth_runner.rb

```diff
@@ -1,15 +1,17 @@
 require 'colored'
 require 'credentials_manager/appfile_config'
 require 'yaml'
+require 'fastlane_core'
 
 require_relative 'tunes/tunes_client'
 
 module Spaceship
   class SpaceauthRunner
-    def initialize(username: nil)
+    def initialize(username: nil, copy_to_clipboard: nil)
```

Comment:
```suggestion
    def initialize(username: nil, copy_to_clipboard: false)
```

Reply:
That would change the default behaviour...

Reply:
☝️ Yeah, here the absence of the param behaves differently than an explicit `false` value